admin/index.php

```
@@ -960,6 +970,9 @@ public static function wp_slimstat_enqueue_scripts($_hook = '')
             'refresh_interval'  => intval(wp_slimstat::$settings['refresh_interval']),
             'page_location'     => self::$page_location,
             'clear_cache_nonce' => wp_create_nonce('slimstat_clear_cache'),
+            // Shared with the filter form-builder so value-less operators (is_empty/
+            // is_not_empty) are never treated as a "remove filter" signal. See #305.
+            'valueless_operators' => wp_slimstat_db::$valueless_operators,
```

PHPStan (2.2.1)

[error] Access to an undefined static property wp_slimstat_db::$valueless_operators.


(staticProperty.notFound)